cairo/src/runner.rs

```diff
@@ -310,21 +513,32 @@ impl<'a, F: Field> CairoProgram<'a, F> {
             mem,
             ini: CairoState::new(F::from(pc), F::from(ap), F::from(ap)),
             fin: CairoState::new(F::zero(), F::zero(), F::zero()),
+            trace: Vec::new(),
         };
         prog.execute();
         prog
     }
 
     /// Outputs the total number of steps of the execution carried out by the runner
-    pub fn get_steps(&self) -> F {
+    pub fn steps(&self) -> F {
         self.steps
     }
 
     /// Outputs the final value of the pointers after the execution carried out by the runner
```

Comment:
Fix comment.  Should be "the initial values of the ..."

Reply:
thanks!